src/components/common/ratio/Ratio.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import "twin.macro";
+import "styled-components/macro";
+import Quantity from "../quantity/Quantity";
+import { CQL } from "cqm-models";
+
+interface RatioProps {
+  label: string;
+  numerator: CQL.Quantity;
+  denominator: CQL.Quantity;
+  canEdit: boolean;
+}
+
+const Ratio = ({ label, numerator, denominator, canEdit }: RatioProps) => {
```

Comment:
Also can we rename this component to `RatioInput` or similar to make sure there is separation from the cql-execution datatypes?

Reply:
Sure

Reply:
done.